mock-core-configs/etc/mock/templates/circlelinux-9.tpl

```diff
@@ -0,0 +1,251 @@
+config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils redhat-release findutils gawk glibc-minimal-langpack grep gzip info patch redhat-rpm-config rpm-build sed tar unzip util-linux which xz'
+config_opts['dist'] = 'el9'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '9'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'quay.io/cclinux/circlelinux:9'
```

Comment:
$ podman run --rm -ti quay.io/cclinux/circlelinux:9 bash
Trying to pull quay.io/cclinux/circlelinux:9...
Error: initializing source docker://quay.io/cclinux/circlelinux:9: reading manifest 9 in quay.io/cclinux/circlelinux: manifest unknown

Reply:
Hey , sorry too late , I just had a long holiday for the Chinese New Year.  now we had pushed podman images to https://quay.io/repository/cclinux/circlelinux ;